src/components/biggive-campaign-card-filter-grid/biggive-campaign-card-filter-grid.tsx

```diff
@@ -18,6 +18,19 @@ export type sortOptionLabel = (typeof sortOptionLabels)[sortOptionKey];
   shadow: true,
 })
 export class BiggiveCampaignCardFilterGrid {
+  private sortByPlaceholderText = 'Sort by';
```

Comment:
Just moving non-annotated properties to be together

Reply:
Fine, but a couple of these I looked at - sortByPlaceholderText and beneficiariesPlaceHolderText only have one usage each so I'm not sure why we ever made them class properties. IMHO would be better to inline them at point of usage.